src/prefect/agent/agent.py

```diff
@@ -554,7 +554,10 @@ def query_flow_runs(self) -> list:
                                     },
                                 },
                             ],
-                        }
+                        },
+                        "order_by": {
```

Comment:
Hmmm couldn't we just as easily do this client-side?  E.g., make the original query w/ an added `scheduled_start_time` attribute and then do a sort on `scheduled_start_time` in this method?